x-pack/plugins/uptime/public/components/monitor/__tests__/monitor_title.test.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import moment from 'moment';
+import * as reactRouterDom from 'react-router-dom';
+import * as redux from 'react-redux';
+import { Ping } from '../../../../common/runtime_types';
+import { MonitorPageTitle } from '../monitor_title';
+import { renderWithRouter } from '../../../lib';
+
+jest.mock('react-router-dom', () => {
+  const originalModule = jest.requireActual('react-router-dom');
+
+  return {
+    ...originalModule,
+    useParams: jest.fn(),
+  };
+});
+
+jest.mock('react-redux', () => {
+  const originalModule = jest.requireActual('react-redux');
+
+  return {
+    ...originalModule,
+    useSelector: jest.fn(),
+  };
+});
```

Comment:
i wonder if it make sense to use mocks we have in file 

`x-pack/plugins/uptime/public/lib/helper/test_helpers.ts`

Reply:
Thanks! This works great. I tried to create one for `react-router-dom` too, but it wouldn't let me use a spy without first running jest.mock in the test file, making a reusable function less helpful.